fdm-core/src/db/schema-custom-types.ts

```
@@ -24,34 +24,296 @@ export const numericCasted = customType<{
+}
+
+function readMultiPoint(
+    dataView: DataView,
+    littleEndian: boolean,
+    offset: number,
+): GeoJSON.Position[] {
+    if (offset + 4 > dataView.byteLength) {
+        throw new Error("Buffer too small to read MultiPoint")
+    }
+    const numPoints = dataView.getUint32(offset, littleEndian)
+    offset += 4
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)